src/Makefile

```
@@ -32,8 +32,17 @@ LUA_LIB:=-llua5.4
 LUA_BIN?=lua5.4
 GRPC_DIR:=../lib/grpc-interfaces
 
-PROTOC:=$(BUILDDIR)/bin/protoc
-GRPC_CPP_PLUGIN=$(BUILDDIR)/bin/grpc_cpp_plugin
+PROTOC:=$(shell which protoc)
+GRPC_CPP_PLUGIN=$(shell which grpc_cpp_plugin)
```

This will call which everytime GRPC_CPP_PLUGIN gets evaluated. Better change to:
```suggestion
GRPC_CPP_PLUGIN:=$(shell which grpc_cpp_plugin)
```

Same for the other entries below: `=` -> `:=`.